packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-embedded/main/index.md

```diff
@@ -0,0 +1,151 @@
+* Configure the Interaction Code grant type in your Okta org.
+* Open and review the sample mobile iOS app from the SDK.
+* Add dependencies and configure the sample mobile app to use Okta.
+* Test your integration by signing in as a user.
+
+#### Sample code
+
+<StackSnippet snippet="samplecode" />
+
+---
+
+## Overview of the Okta mobile sign-in flow
+
+This guide introduces using the Okta Client SDK for Swift for integrating authentication flows in to your browserless, mobile apps.
+
+You initialize your mobile app with your Okta org's app integration details. Then the flow requests the initial step, and then cycles through responding to steps until the user signs in, cancels, or an error occurs. Each sign-in step can include one or more user actions, such as choosing an authenticator or entering a one-time passcode (OTP).
+
+The following sections demonstrate the iOS sign-in flow processes using`okta-mobile-swift` SDK, and begins by setting up your Okta org integration and then the SDK sample app ([IdxAuthSignIn](https://github.com/okta/okta-mobile-swift/tree/master/Samples/IdxAuthSignIn#idxauthsignin)).
+
+## Set up your Okta org
+
+Set up your [Okta org](/docs/concepts/okta-organizations/). You can sign up free for an [Okta Integrator Free Plan org](https://developer.okta.com/signup/) that works with the sample app.
+
+Make a note of your Okta domain. Use it wherever `{yourOktaDomain}` appears in this guide.
+
+### Enable the Interaction Code grant type for your org
+
+Enable this grant type to allow apps the ability to use embedded sign-in flows across your entire org. You must have super admin permissions to enable this setting.
+
+1. Open the Admin Console for your org.
+1. Go to **Settings** > **Account** > **Embedded widget sign-in support**.
+1. Click **Edit**.
+1. Click **Advanced** in the **Grant type** section, and then select **Interaction Code**.
+1. Click **Save**.
+
+For further information on the Interaction Code grant type, see [Interaction Code grant type](/docs/concepts/interaction-code/).
+
+### Create an Okta integration for your iOS app
+
+An app integration represents your app in your Okta org. The integration configures how your app integrates with the Okta services. This includes which users and groups have access, authentication policies, token refresh requirements, redirect URLs, and more. The integration includes configuration information required by the app to access Okta.
+
+To create your app integration in Okta using the Admin Console:
+
+1. [Sign in to your Okta organization](https://developer.okta.com/login) with your administrator account. Click **Admin** on the top right of the page.
+1. Open the apps configuration page by selecting **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. Select a **Sign-in method** of **OIDC - OpenID Connect**.
+1. Select an **Application type** of **Native Application**, then click **Next**.
```

Comment:
Instead of “Select an Application type of Native Application…”, how about  
**“Select ‘Native Application’ as the application type, then click Next.”**